winui/AutoComplete/Leading-and-Trailing-view.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Leading and trailing view in WinUI AutoComplete | Syncfusion
+description: Learn how to add leading and trailing view (front and back view) in Syncfusion WinUI AutoComplete control and more.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# Leading and Trailing View in WinUI AutoComplete (SfAutoComplete)
+
+This section explains about the leading and trailing view support available in [AutoComplete](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfAutoComplete.html).The `LeadingView` appears before the `AutoComplete` selection area and `TrailingView` appears after the `AutoComplete` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView`.

```

Comment:
This section explains about the leading and trailing view support available in [AutoComplete](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfAutoComplete.html). The `LeadingView` appears before the `AutoComplete` selection area and `TrailingView` appears after the `AutoComplete` selection area. Any content, such as an icon, image, button, or control, can be displayed in the `LeadingView` and `TrailingView.`